### PR TITLE
Added check for design.zip existence

### PR DESCRIPTION
**Pull request by inestyne**
fstat errors out when file does not exist. Fixed this, put 2 seperate error messages, and made the other function in update.js dependent upon successfully unzipping the file

### CODE REVIEW

File: lib/update/unzip-design.js [hunk 1]
```diff
@@ -8,15 +8,22 @@ module.exports = function unzipDesign() {
   try {
     const hasZipFile = fs.statSync(DESIGN_FILE).isFile();
     if (!hasZipFile) {
-      return;
+      console.log(`Error: Cannot stat ${DESIGN_FILE} file\nPlease re-download your webflow zip file and copy it here, renaming it to design.zip.`);
```
Comment (Chun-Yang) [inline lib/update/unzip-design.js]: Should be 
```
Cannot find
```
Right?

**Comment by inestyne** [comment]
Hehe, forgot to run lint

File: lib/update/index.js [hunk 2]
```diff
@@ -5,13 +5,14 @@ const copyFonts = require('./copy-fonts');
 const runReacterminator = require('./run-reacterminator');
 
 module.exports = function update() {
-  unzipDesign();
```
Comment (Chun-Yang) [inline lib/update/index.js]: How about we do not change this file? Instead of checking the return value, we rely on the error to stop the program?

**Comment by Chun-Yang** [comment]
Hey @inestyne I just remembered stanza can also use the `.design` folder instead of the zip file. I will take care of this feature.

Sorry about not able to merge the PR.

Thank you very much!

**Pull request closed without merging**